scripts/2022_10_03_67_set_votes_for_week_10_06__10_13.py

```diff
@@ -0,0 +1,126 @@
+import math
+
+from ape_safe import ApeSafe
+from brownie import Contract, accounts, network
+from helpers import (
+    CHAIN_IDS,
+    GAUGE_ABI,
+    GAUGE_CONTROLLER_ADDRESS,
+    MULTISIG_ADDRESSES,
+    SDL_ADDRESSES,
+    SDL_DAO_COMMUNITY_VESTING_PROXY_ADDRESS,
+    SDL_MINTER_ADDRESS,
+)
+
+from scripts.utils import confirm_posting_transaction
+
+TARGET_NETWORK = "MAINNET"
+
+
+def main():
+    """
+    Set Gauge weights for week 10_06_2022 -> 10_20_2022 from results of snapshot vote
+    Vote: https://snapshot.org/#/saddlefinance.eth/proposal/0xf6f188d0b96a7db164c82bc946f26247888dc7464927c025d43dc0aa0bd5074b
+    """
+
+    print(f"You are using the '{network.show_active()}' network")
+    assert network.chain.id == CHAIN_IDS[TARGET_NETWORK], f"Not on {TARGET_NETWORK}"
+    multisig = ApeSafe(MULTISIG_ADDRESSES[CHAIN_IDS[TARGET_NETWORK]])
+    sdl = multisig.contract(SDL_ADDRESSES[CHAIN_IDS[TARGET_NETWORK]])
+    sdl_vesting_contract_proxy = multisig.contract(
+        SDL_DAO_COMMUNITY_VESTING_PROXY_ADDRESS[CHAIN_IDS[TARGET_NETWORK]]
+    )
+    deployer = accounts.load("deployer")  # prompts for password
+
+    # Release vested tokens to multisig account
+    sdl_vesting_contract_proxy.release()
+
+    gauge_controller = multisig.contract(GAUGE_CONTROLLER_ADDRESS[CHAIN_IDS["MAINNET"]])
+
+    # gauges to add to gauge controller
+    gauge_data = {
+        "SaddleFRAXBPVesperFRAXMetaPoolGauge": {
+            "address": "0x9980C9b35844946cF3451cC0B43D9b6501F4a96E",
+            "type": 0,
+            "weight": 0,
+        },
+        "SaddleFRAXWCUSDMetaPoolGauge": {
+            "address": "0x3dC88ee38db8C7b6DCEB447E4348e51bd87ced93",
```

Comment:
Script fails on forked mainnet due to this gauge is already added to our gauge controller.